includes/admin/settings/register-settings.php

```diff
@@ -569,8 +569,9 @@ function edd_get_registered_settings() {
 						'name'  => __( 'Usage Tracking', 'easy-digital-downloads' ),
 						'check' => __( 'Allow',          'easy-digital-downloads' ),
 						'desc'  => sprintf(
-							__( 'Help us make Easy Digital Downloads better by opting into anonymous usage tracking. <a href="%s" target="_blank">Here is what we track</a>.<br>If you opt-in here and to <a href="%s">our newsletter</a>, we will email you a discount code for our <a href="%s" target="_blank">extension shop</a>.', 'easy-digital-downloads' ),
-							esc_url( 'https://easydigitaldownloads.com/tracking/' ),
+							/* translators: %1$s Link to tracking information, do not translate. %2$s Link to EDD newsleter, do not translate. %3$s Link to EDD extensions, do not translate */
+							__( 'Help us make Easy Digital Downloads better by opting into anonymous usage tracking. <a href="%1$s" target="_blank">Here is what we track</a>.<br>If you opt-in here and to <a href="%2$s">our newsletter</a>, we will email you a discount code for our <a href="%3$s" target="_blank">extension shop</a>.', 'easy-digital-downloads' ),
```

Comment:
While we're here, let's make the newsletter link open in a new tab like the others do.